odoo/tools/_monkeypatches.py

```diff
@@ -12,6 +12,12 @@
 
 from .json import scriptsafe
 
+try:
+    from stdnum import util
+except ImportError as e:
+    util = None
+    _logger.warning("Failed to import stdnum library: %s", e)
```

Comment:
I don't think this log is very useful.

`stdnum` is part of the requirement, so in most cases, you won't ever see it.
In case you don't fulfill the requirement, it's either that you forgot to install them and it will crash anyways, or that you're not fulfilling them on purpose (light deployment such as IoT box & co), in which case a warning on every start up is just noise.